### ISSUE TITLE
Crate sudoku

### ISSUE THREAD

**Issue by ken1000minus7** [PEC-CSS/acm-fun#11]
Upto your imagination how u want to make

**Comment by YashitaBansal** [comment]
I am interested. Please assign it to me.

**Comment by ken1000minus7** [comment]
@YashitaBansal any updates?

**Comment by YashitaBansal** [comment]
> @YashitaBansal any updates?

Yes sir working on it, there are alot of conflicts so sir but I will try to do it by tomorrow sir.
